### PR TITLE
Revamp UI admin

### PR DESCRIPTION
**Pull request by revengemiroz**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

## Showcase

Please attach a picture or a video showing what this change does.

### CODE REVIEW

File: apps/admin/src/pages/Home/index.tsx [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 function Home() {
-  const { data } = useGetAllProjectsQuery();
+  // const { data } = useGetAllProjectsQuery();
 
   return (
-    <div>
-      <h2>Projects</h2>
-
+    <div className='bg-gray-200'>
+      {/*
```
Comment (rohilpinto) [inline apps/admin/src/pages/Home/index.tsx:12]: can you

File: apps/admin/src/pages/Home/index.tsx [hunk 2]
```diff
@@ -1,21 +1,24 @@
 import React from 'react';
 
-import { useGetAllProjectsQuery } from 'apollo-hooks';
+import { NextSeo } from 'next-seo';
+
+// import { useGetAllProjectsQuery } from 'apollo-hooks';
```
Comment (rohilpinto) [inline apps/admin/src/pages/Home/index.tsx]: could you please delete this comment

File: apps/admin/src/pages/Home/index.tsx [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
 
 function Home() {
-  const { data } = useGetAllProjectsQuery();
+  // const { data } = useGetAllProjectsQuery();
```
Comment (rohilpinto) [inline apps/admin/src/pages/Home/index.tsx]: could you please delete this comment

File: apps/admin/src/pages/Home/index.tsx [hunk 4]
```diff
@@ -1,21 +1,24 @@
...
-          <div key={project.id}>
-            <p>{project.title}</p>
-          </div>
+      {data?.projects.results?.map((project) => (
+        <div key={project.id}>
+        <p>{project.title}</p>
+        </div>
         ))}
-      </div>
+      </div> */}
```
Comment (rohilpinto) [inline apps/admin/src/pages/Home/index.tsx]: could you please delete this comment

**Review by rohilpinto — CHANGES_REQUESTED** [review]
Requesting some comment deletions

File: apps/admin/src/pages/Home/index.tsx (hunk 4 above)
Comment (revengemiroz) [reply apps/admin/src/pages/Home/index.tsx]: done

File: apps/admin/src/pages/Home/index.tsx (hunk 3 above)
Comment (revengemiroz) [reply apps/admin/src/pages/Home/index.tsx]: done

File: apps/admin/src/pages/Home/index.tsx (hunk 2 above)
Comment (revengemiroz) [reply apps/admin/src/pages/Home/index.tsx]: done

**Review by ivanms1 — APPROVED** [review]
LGTM

**Pull request merged**